code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -1060,13 +1060,13 @@
 	name = "Capitalist Golem"
```

Comment:
it'd also be nice if we could change the racial name "capitalist golem" to "capitalism golem", but eh, that might be a lot of effort for not much gain

Reply:
Nah, that's an easy change.

Reply:
I changed the money golems to Capitalism Golems and the russian ones to Socialism Golems.